code/citadel/dogborgs.dm

```diff
@@ -70,6 +70,7 @@
 	modules += new /obj/item/soap/tongue(src)
 	modules += new /obj/item/device/healthanalyzer(src)
 	modules += new /obj/item/dogborg/sleeper(src)
+	modules += new /obj/item/reagent_containers/borghypo(src)
```

Comment:
This file is more or less kept as a reference to 5th port. since stuff's modularized out, you don't need to add code to this.